src/acp/control.ts

```diff
@@ -116,29 +116,29 @@ export type WithLinkedAcpAccessControl<
  * function is still experimental and subject to change, even in a non-major release.
  * ```
  *
- * Initialise a new [[AccessControl]].
+ * Initialise a new [[Control]].
  */
-export function createAccessControl(
+export function createControl(
   options?: Parameters<typeof createThing>[0]
-): AccessControl {
-  let accessControl = createThing(options);
-  accessControl = setIri(accessControl, rdf.type, acp.AccessControl);
-  return accessControl;
+): Control {
+  let control = createThing(options);
+  control = setIri(control, rdf.type, acp.AccessControl);
+  return control;
 }
 /**
  * ```{note} The Web Access Control specification is not yet finalised. As such, this
  * function is still experimental and subject to change, even in a non-major release.
  * ```
  *
- * Find an [[AccessControl]] with a given URL in a given Resource with an Access Control Resource.
+ * Find an \[\[Control\]\] with a given URL in a given Resource with an Access Control Resource.
```

Comment:
Did something go wrong here with a search and replace? The same happens at a couple of other places below.

```suggestion
 * Find an [[Control]] with a given URL in a given Resource with an Access Control Resource.
```

Reply:
Exactly ^^. Fixed in c9d70605d4b499bd647259fce5594bef2c8a4718